administrator/templates/khonsu/index.php

```diff
@@ -65,7 +65,44 @@
 		opacity: 0;
 	}
 ';
-// $this->addStyleDeclaration($css);
+$scroll ='
```

Comment:
This should not be used. It is not supported on all browsers. 

This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

Reply:
okay thank you sir